index.html

```diff
@@ -6,6 +6,9 @@
 <title>Spirit Journey Formation Anniversary</title>
 <meta name="viewport" content="width=device-width, initial-scale=1">
 <link rel="stylesheet" href="css/main.css">
+<script defer src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.18.1/moment.min.js"></script>
+<script defer src="https://cdnjs.cloudflare.com/ajax/libs/humanize-duration/3.10.0/humanize-duration.min.js"></script>
+<script defer src="js/listdates-min.js"></script>
```

Comment:
I like this movement. Also, I didn't know about the difference between async and defer, thank you for prompting me to research! http://www.growingwiththeweb.com/2014/02/async-vs-defer-attributes.html

I don't even know how async would work without a guarantee of orderliness. Later research I guess.